feature/#508 change Pipeline and PipelineConfig documentation (#535)
Avaiga/taipy-doc: docs/manuals/core/concepts/scenario.md

problem with different assumptions. This is extremely useful in a business context where impact analysis
and what-if analysis are essential in the decision process.

> A scenario contains an executable Directed Acyclic Graph (or DAG). The scenario DAG is a set of tasks connecting

Reviewer: ```suggestion
A scenario contains a Directed Acyclic Graph (or DAG) submittable for execution. The scenario DAG is a set of tasks connecting
```

> data nodes together. It can also be broken down to smaller graphs for execution by defining a `Sequence^`.

Reviewer 2: It can also be broken down to smaller graphs for execution by defining `Sequence^`s.

> data nodes together. It can also be broken down to smaller graphs for execution by defining a `Sequence^`.

Reviewer 3: ```suggestion
data nodes together. It can also be broken down into smaller graphs for execution by defining a `Sequence^`.
```

> A sequence is a subset of tasks derives from the scenario's set of tasks, forming a smaller executable DAG that

Reviewer 2: "derives" -> "derived"

> A sequence is a subset of tasks derives from the scenario's set of tasks, forming a smaller executable DAG that

Reviewer 3: ```suggestion
A sequence is a subset of tasks derived from the scenario's set of tasks, forming a smaller executable DAG that
```

> A sequence is a subset of tasks derived from the scenario's set of tasks, forming a smaller executable DAG that

Reviewer: ```suggestion
A sequence is a subset of connected tasks derived from the scenario's set of tasks, forming a smaller executable DAG that
```

can be submitted separately from the scenario DAG. A scenario can also contain a set of other data nodes

> outside of the scenario DAG to represent additional data related to the scenario but are not executable.

Reviewer: ```suggestion
disconnected from the scenario DAG to represent additional data related to the scenario. The execution of the scenario does not compute the other data nodes.
```

Author: Have fixed it to  "A scenario can also contain 
a set of additional data nodes that are not part of the scenario DAG to represent additional data related to 
the scenario. The execution of the scenario does not compute the additional data nodes." instead. What do you think?

Reviewer: that s great!

> outside of the scenario DAG to represent additional data related to the scenario but are not executable.

Reviewer 2: What is the need for the "but are not executable" fragment?

Author: @FabienLelaquais The additional DNs are for data that are related to the scenario but are not part of the DAG, so they are not submittable. Hence not executable 😃

Reviewer 2: Of course they are not. That's my point. Why even mention it.

Author: Hmm I wanted to make it clear that it doesn't belong to the DAG as we did mention before that, a DAG is a collection of tasks and datanodes. So I'm afraid user might be confused if they should provide the data nodes again aside from the task for the DAG

Reviewer 2: What I meant is that indeed, tasks are executable. In a sense, sequences and scenarios are as well. 
But unless you understood nothing, data nodes never will be.

The addition of "[data nodes] [...] are not executable" really seems strange here. Is similar to saying: "Oh by the way, giraffes cannot talk.".

Author: ah I see what you meant, then how about "A scenario can also contain a set of additional data nodes that are not part of the scenario DAG to represent additional data related to the scenario."?

> outside of the scenario DAG to represent additional data related to the scenario but are not executable.

Reviewer 3: ```suggestion
outside of the scenario DAG to represent other data related to the scenario but are not executable.
```

After analyzing its first scenario, an end-user may be interested in modifying input data nodes

> (not the intermediate nor the output data nodes), re-running the same sequences or scenario, and

Reviewer 3: ```suggestion
(not the intermediate nor the output data nodes), re-running the identical sequences or scenario and
```

!!! example "In the example"

>     Here, our scenario consists of the a graph of data nodes and tasks. The scenario graph can also be split into

Reviewer 3: ```suggestion
    Here, our scenario consists of a graph of data nodes and tasks. The scenario graph can also be split into
```

    two sequences as subgraphs described earlier. The external light blue box in the flowchart below represents

>     our scenario that contains all data nodes, tasks and sequences.

Reviewer 3: ```suggestion
    our scenario that contains all data nodes, tasks, and sequences.
```
